app/modules/vm/index.coffee

```diff
@@ -472,11 +475,17 @@ module.exports = angular.module 'xoWebApp.vm', [
 
         return
 
+      # Disk resize
+      forEach disks, (attributes, id) ->
+        disk = get id
+        if attributes.size isnt disk.size
+          promises.push (xo.disk.resize id, attributes.size)
```

Comment:
Please, use parentheses around `xo.disk.resize()`'s because relying on the space between `promises.push` and the first parenthesis is hard and prone to future error.